content/webapp/components/FeaturedCard/FeaturedCard.styles.ts

```diff
@@ -26,8 +24,7 @@ export const FeaturedCardLink = styled.a.attrs({
   className: 'grid',
   'data-gtm-trigger': 'featured_card_link',
 })<HasIsReversed>`
-  justify-content: flex-end;
-  flex-direction: ${props => (props.$isReversed ? 'row-reverse' : 'row')};
+  gap: 0 !important;
```

Comment:
TODO: maybe do this with a .no-gap class to avoid having to add !important?

Reply:
I'll leave this for now. I think maybe the better direction to go in would be replacing all instances of <div className="grid"> with a `<Grid $gap={x} />` Styled-component which would also solve this

Reply:
I see you're referring to it here! Sorry! I think that'd be good to do 👍

Reply:
Made the `Grid` styled component with a `$noGap` prop